package-lock.json

```diff
@@ -952,14 +953,14 @@
     },
     "autoprefixer": {
       "version": "7.2.6",
-      "resolved": "https://registry.npmjs.org/autoprefixer/-/autoprefixer-7.2.6.tgz",
-      "integrity": "sha512-Iq8TRIB+/9eQ8rbGhcP7ct5cYb/3qjNYAR2SnzLCEcwF6rvVOax8+9+fccgXk4bEhQGjOZd5TLhsksmAdsbGqQ==",
+      "resolved": "https://artifactory.ciena.com/api/npm/blueplanet-npm/autoprefixer/-/autoprefixer-7.2.6.tgz?dl=https://registry.npmjs.org/autoprefixer/-/autoprefixer-7.2.6.tgz",
```

Comment:
This is incorrect. Since this is a public repo you do not want it to point at `artifactory`. To fix this you will need to remove the setting in your dev environment that is setting the `registery` to `artifactory` for everything and instead place a `.npmrc` file with that `registry` setting in your directories for private repos only.